nix/caf/source.json

```diff
@@ -1,7 +1,7 @@
 {
-  "owner": "tenzir",
+  "owner": "actor-framework",
   "repo": "actor-framework",
-  "rev": "4ff66bf218dec0514829230af6f41fb8ed09aa5c",
-  "sha256": "s7CdmQ64whJ7o7KiIxKLrZSR4RRJnNzcgGuqGKcxCzI=",
-  "version": "0.18.7"
+  "rev": "1.0.2",
+  "hash": "sha256-1DJ8VYBTC4Kd6IQZoj4AjP3CoHhb+bmtBEozc5T0R/0=",
```

Comment:
Just for my understanding: What's the difference between specifying `sha256` and `hash` for Nix here?

Reply:
This is simply a migration that the nix ecosystem has been doing. Functionally the two arguments are equivalent.

Technically: `hash` takes an sri encoded hash, which specifies the function as part of the string. That way we don't need varargs to support all the different hash functions in wrappers that ultimately boil down to calls to `builtins.derivation` any more.

Reply:
Thanks for the explanation 🙏